syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -29,7 +38,12 @@ class Syft internal constructor(
     companion object {
         @Volatile
         private var INSTANCE: Syft? = null
-
+        
+        /**
+        * Function to get the instance of the Syft Worker.
```

Comment:
Singleton syft worker instance. Only one syft worker should be running at a time in an application